packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -108,7 +107,8 @@ export function TimeRangePicker(props: TimeRangePickerProps) {
 
   const styles = useStyles2(getStyles);
   const { modalBackdrop } = useStyles2(getModalStyles);
-  const hasAbsolute = isDateTime(value.raw.from) || isDateTime(value.raw.to);
+  const hasAbsolute = !rangeUtil.isRelativeTime(value.raw.from) || !rangeUtil.isRelativeTime(value.raw.to);
```

Comment:
This change is required, as in the old arch range.raw.to/from could have been expressed as moment object or string. In Scenes we guarantee it's always string.